src/components/Counter.js

```diff
@@ -6,34 +6,41 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        if(this.props.count % 2 != 0){
+            this.props.increment()
+        }
+
     };
 
     incrementAsync = () => {
         // Stretch Problem: Implement an increment function that
         // increments after waiting for one second
+        setInterval(this.props.increment,1000)
```

Comment:
```suggestion
        setTimeout(this.props.increment, 1000)
```
Setting interval will continue running this.props.increment every 1 second forever. The task here was to increment it 1 time after 1 second.